Source/WTF/Configurations/WTF.xcconfig

```diff
@@ -42,3 +42,7 @@ ICU_HEADERS_FOLDER_PATH_NO = $(BUILT_PRODUCTS_DIR)$(WK_LIBRARY_HEADERS_FOLDER_PA
 ICU_HEADERS_FOLDER_PATH_YES = $(WTF_INSTALL_PATH_PREFIX)$(WK_LIBRARY_HEADERS_FOLDER_PATH)/unicode;
 
 SYSTEM_FRAMEWORK_SEARCH_PATHS = $(inherited) $(WK_PRIVATE_SDK_DIR)$(SYSTEM_LIBRARY_DIR)/PrivateFrameworks
+
+// Used to generate headers filelist for TAPI in JavaScriptCore. CPP_HEADERMAP_PRODUCT_HEADERS_VFS_FILE
+// is not exported to script phases, so it must be bound to a different name.
+WK_HEADERS_VFS_FILE = $(CPP_HEADERMAP_PRODUCT_HEADERS_VFS_FILE);
```

Comment:
Why don't we just use the CPP_HEADERMAP_PRODUCT_HEADERS_VFS_FILE environment variable directly?

Reply:
I observed that it wasn't being exported to script phases. Didn't dive deep enough to figure out exactly why, but there are a class of build settings that are not automatically exported, so I assume this is one of them.